packages/database-editor/src/components/ImportWizard/pages/CreationPage.tsx

```diff
@@ -0,0 +1,67 @@
+import type { DatabaseTable } from '@axonivy/database-editor-protocol';
+import { Checkbox, Flex, Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from '@axonivy/ui-components';
+import { useTranslation } from 'react-i18next';
+import './CreationPage.css';
+
+export type CreationParameter = {
+  tableName: string;
+  enum?: boolean;
+  entity?: boolean;
+  repository?: boolean;
+};
+
+export type CreationPageProps = {
+  tables: Array<DatabaseTable>;
+  parameters: Array<CreationParameter>;
+  updateSelection: (table: string, key: Exclude<keyof CreationParameter, 'tableName'>, value: boolean) => void;
+};
+
+export const CreationPage = ({ tables, updateSelection, parameters }: CreationPageProps) => {
+  const { t } = useTranslation();
+
+  const checkState = (tableName: string, key: Exclude<keyof CreationParameter, 'tableName'>): boolean => {
+    const param = parameters.find(p => p.tableName === tableName);
+    if (!param) return false;
+    return param[key] === true;
```

Comment:
I'm not the biggest fan of onliner if statements, even if sometimes they are useful.
Here however I don't get this idea...
```suggestion
    return param && param[key] === true;
```